apps/gp2-frontend/src/working-groups/WorkingGroupDetail.tsx

```diff
@@ -10,10 +10,17 @@ import { NotFoundPage } from '@asap-hub/react-components';
 import { useCurrentUserGP2 } from '@asap-hub/react-context';
 import { gp2 as gp2Routing, useRouteParams } from '@asap-hub/routing';
 import { Redirect, Route, Switch } from 'react-router-dom';
+import EditResource from './EditResource';
 import { usePutWorkingGroupResources, useWorkingGroupById } from './state';
 
 const { workingGroups } = gp2Routing;
 
+const modalInfo = {
+  title: 'Add Resource',
+  description:
+    'Select a resource type and provide the neccessary information required to share a resource privately with your group.',
+};
```

Comment:
Add Resource? This is editing a resource? 

Should we have tests asserting the title and description are correct?

Reply:
Done